XpringKit/XRP/Model/XRPEscrowCancel+Org_Xrpl_Rpc_V1_EscrowCancel.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/// Conforms to XRPEscrowCancel struct while providing an initializer that can construct an XRPEscrowCancel
+/// from an Org_Xrpl_Rpc_V1_EscrowCancel
+internal extension XRPEscrowCancel {
+
+  /// Constructs an XRPEscrowCancel from an Org_Xrpl_Rpc_V1_EscrowCancel
+  /// - SeeAlso: [EscrowCancel Protocol Buffer]
+  /// (https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/
+  /// src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L170)
+  ///
+  /// - Parameters:
+  ///     - escrowCancel: an Org_Xrpl_Rpc_V1_EscrowCancel (protobuf object) whose field values will be used to
+  ///             construct an XRPEscrowCancel
+  /// - Returns: an XRPEscrowCancel with its fields set via the analogous protobuf fields.
+  init?(escrowCancel: Org_Xrpl_Rpc_V1_EscrowCancel, xrplNetwork: XRPLNetwork) {
+    guard let ownerXAddress = Utils.encode(
+      classicAddress: escrowCancel.owner.value.address,
+      isTest: xrplNetwork != XRPLNetwork.main
```

Comment:
worth adding a helper like i did? i kno u mentioned it in my pr but was curious if you're gonna do that b4/after the wrappers are done

Reply:
+1 on this being a good idea. 

You could just add it as an extension on the XRPLNetwork enum. 

```swift
extension XRPLNetwork {
  public var isTest: Bool {
     return self != .Main
  }
}
```

Reply:
Yep - going to actually do this all at once when all the wrappers are in